docs/source/tutorial/schema.md

```diff
@@ -204,11 +204,41 @@ server.listen().then(({ url }) => {
 
 After saving, run `npm start` to start your server! 🎉 Apollo Server is now available on port 4000.
 
-### Explore your schema
+## Explore your schema
 
-With your server running, visit `localhost:4000` in your browser to open [GraphQL Playground](https://www.apollographql.com/docs/apollo-server/features/graphql-playground/), an IDE that enables you to **introspect** your schema and test out queries. Introspection is a helpful feature of a GraphQL server that enables you to obtain its schema.
+Now that our server is running, we can **introspect** its schema types and fields with useful developer tools. Later, we'll also be able to run test queries from those tools.
 
-> Note that introspection should be **disabled** for a production GraphQL server. Apollo Server disables introspection automatically if the `NODE_ENV` environment variable is set to `production`.
+> Introspection is a helpful GraphQL feature that enables you to obtain a server's schema for development purposes. It should be **disabled** for servers running in production. Apollo Server disables introspection automatically if the `NODE_ENV` environment variable is set to `production`.
+
+### With Apollo Studio (recommended)
+
+The **Apollo Studio Explorer** is a powerful free web IDE for exploring your GraphQL schema and building queries against it:
+
+<img class="screenshot" src="../img/explorer-tab.jpg" />
+
+To use the Explorer with your locally running server, you register a **dev graph** with Apollo Studio, like so:
+
+1. Visit [studio.apollographql.com/dev](https://studio.apollographql.com/dev).
+
+    You'll be prompted to create an Apollo account if you don't already have one (again, Apollo accounts and the Explorer are completely free).
+
+2. In the form that appears, specify a name for your graph and your local server's URL (`http://localhost:4000` for this tutorial):
+
+    <img class="screenshot" src="../img/dev-graph.jpg" width="450" alt="Dev graph form" />
+
+3. Click **Create graph**. Studio automatically opens the Explorer to use with your server!
+
+To introspect your server's schema in the Explorer, click the Documentation tab on the left side:
+
+<img class="screenshot" src="../img/explorer-documentation-tab.jpg" alt="Explorer documentation tab" width="200"/>
+
+Your schema's root types (`Query` and `Mutation`) appear. You can click any type to view its fields and step down into your schema. You'll see that the types and fields match what we provided to our server.
+
+> [Learn more about the Apollo Studio Explorer](https://www.apollographql.com/docs/studio/explorer/)
+
+### With GraphQL Playground
+
+With your server running, visit `localhost:4000` in your browser to open [GraphQL Playground](https://www.apollographql.com/docs/apollo-server/features/graphql-playground/).
```

Comment:
I wonder if we might want to say something about this being a good option for simple needs, but lacking a substantial number of the features/tools in the Explorer?